src/env.ts

```diff
@@ -125,6 +125,11 @@ const envDefinitions = {
     def: "https://tron.coin.ledger.com",
     desc: "proxy url for trongrid API",
   },
+  API_HEDERA_MIRROR: {
+    def: "https://mainnet-public.mirrornode.hedera.com",
```

Comment:
Using the ledger reverse-proxy here works 👍🏻. I'll push up the edit once I figure out how to set the reverse-proxy in `network.ts` (running into some difficulty using the aforementioned `client.setNetwork()` function).

Reply:
👋🏻 @haammar-ledger

I've tried setting the `hedera.Client` to target `https://hedera.coin.ledger.com`, but it doesn't seem to work. I'm not sure why, but perhaps because `hedera.Client.forMainnet()` sets into the client a network of nodes (which are the mainnet nodes). 

This is the structure/how it looks:
> MAINNET: {
        "35.237.200.180:50211": new AccountId(3),
        "35.186.191.247:50211": new AccountId(4),
        "35.192.2.25:50211": new AccountId(5),
        "35.199.161.108:50211": new AccountId(6),
        "35.203.82.240:50211": new AccountId(7),
        "35.236.5.219:50211": new AccountId(8),
        "35.197.192.225:50211": new AccountId(9),
        "35.242.233.154:50211": new AccountId(10),
        "35.240.118.96:50211": new AccountId(11),
        "35.204.86.32:50211": new AccountId(12),
        "35.234.132.107:50211": new AccountId(13),
        "35.236.2.27:50211": new AccountId(14),
        "35.228.11.53:50211": new AccountId(15),
        "34.91.181.183:50211": new AccountId(16),
        "34.86.212.247:50211": new AccountId(17),
        "172.105.247.67:50211": new AccountId(18),
        "34.89.87.138:50211": new AccountId(19),
        "34.82.78.255:50211": new AccountId(20),
    }

👆🏻 [Source](https://github.com/hashgraph/hedera-sdk-js/blob/f48dcb2bb2a596fae5fca7eea6c76a862a446ee4/src/client/NodeClient.js#L37-L56)

I'm having a difficult time seeing how to target the network directly.. but if you have any suggestions, please do let me know! Otherwise, I hope it will be okay to leave it as is.